src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -468,6 +520,24 @@ class LicenceService(
             eventDescription = "${reason ?: "Licence automatically inactivated"} for ${licence.forename} ${licence.surname}",
           ),
         )
+
+        if (licence.kind == CRD) {
```

Comment:
could remove the conditionals/duplication here by adding  `inactivedDomainEventType` and `activatedDomainEventType` to LicenceKind. 

Need to support `HardStop` Licence types too which would be `LICENCE_ACTIVATED` / `LICENCE_INACTIVATED`

By adopting that pattern it enforces handling every licence type